### ISSUE TITLE
FPP V8: Video does not appear when clip starts, the monitor is not recognizing the video signal. 

### ISSUE THREAD

**Issue by patdelaney** [FalconChristmas/fpp#1825]
**FPP Version and Hardware**: 
FPP 8.x master running in Pi 3+, but is reproducible in Pi 2 as well

**Describe the bug and Steps to reproduce**: 
My theory currently is that when Pi is done playing a video, the HDMI output goes to sleep, which triggers the monitors to go to sleep. I don't remember this behavior in pre 8.x. So when video starts to play the monitors wakeup or re-adjust the the HDMI signal it's getting.

I've tested with 2 different smaller sets of monitors (different manufacturers) with the same results on v8.x
https://www.amazon.com/gp/product/B076GZVCP2/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1

I tested with 6.3 and got the following results:
- It appears that when the video stops playing we are just sending a blank or black signal. The monitor does not go to sleep. So when I start the clips the video is immediately seen.

I did fppos update to 7.5 with latest patches, tested and got the following results:
- Same results as 6.3  (good)

I did fppos update to 8 alpha 3 with latest patches, tested and got the following results:
- The delay is back, I think we are completely turning off video output in 8.x. 
- I turned off the screen saver and it's the same behavior.
- I created a playlist with 3 video clips and it's the same behavior. It appears we are toggling some sort of video playback mode between each clip.

**Comment by darylc** [comment]
@patdelaney Is this still an issue with FPP 8 beta1?

**Comment by patdelaney** [comment]
@darylc at this point yes for only one of my test insances, although I do have 1 FPP that is working correctly and I'm trying to compare settings to understand what the differences are betweek the working and non-working pi's

**Comment by patdelaney** [comment]
Here is the current test scenario:

- 3 Raspberry Pi4s
- 1 master, 2 remotes
- The master is now behaving correctly, but the remotes have the issue described above.

I did compared the cmdline.txt files between the one that works and the remote that doesn't and the only difference was the remote was missing vc4.force_hotplug=0x03. I madded that setting to the remote and rebooted  but it didn't make a difference.
![image](https://github.com/user-attachments/assets/fc9fbeb4-ccd4-4a19-9052-f3b6579f99c1)
